src/core/thread/mle.cpp

```diff
@@ -788,15 +788,20 @@ ThreadError Mle::AppendLeaderData(Message &aMessage)
     return aMessage.Append(&mLeaderData, sizeof(mLeaderData));
 }
 
+void Mle::FillNetworkDataTlv(NetworkDataTlv &aTlv, bool aStableOnly)
```

Comment:
I'm not sure why you need to define a method for populating a TLV object.  For example, it seems that using `Mle::AppendNetworkData()` directly should be sufficient.